INSTALL

```diff
@@ -299,6 +299,9 @@
                    is left enabled (the default), then compression will only
                    work if the zlib or zlib-dynamic options are also chosen.
 
+  no-crmf
```

Comment:
There are standardized uses cases for CRMF other than CMP. RFC 5272, "Certificate management over CMS" utilizes either PKCS#10 or CRMF. 

If it's the strategy to reduce those options (didn't look like that, but I understand that e.g. run-checker takes quite some time), then it would be easy to reduce and have exclusively no-cmp for both for now.

Reply:
I'll remove the mention of the `no-crmf` option from INSTALL.

Since the implementation is in two directories: `crypto/crmf/` and `crypto/cmp/` I presume that we need to mention both inside the `Configure` script and this will lead to potentially both macros: `OPENSSL_NO_CRMF` and `OPENSSL_NO_CMP`. Since moreover `"cmp"` implies `"crmf"` (which is going to be reflected in `Configure`, I also had to replace all `#ifndef OPENSSL_NO_CRMF` 
by `#if !defined(OPENSSL_NO_CRMF) && !defined(OPENSSL_NO_CMP)`